src/components/draw/DrawStyleDirective.js

```diff
@@ -15,16 +15,48 @@ goog.require('ga_window_service');
   module.directive('gaDrawStyle', function($document, $window, $translate,
       gaGlobalOptions, gaStyleFactory, gaUrlUtils, gaWindow) {
 
+    var findCategoryBySource = function(src, categories) {
+      var regexCss = new RegExp('^(.*?)-24');
+      if (regexCss.test(src)) {
+        return categories[0];
```

Comment:
This line bother me . It will not work if we have more than 1 category with css icons. If you find antoher way to make it more generic, fix it otherwise we leave as it is for now.

Reply:
If you don't find a solution add o TOFIX comment. `// TOFIX: this line will nor work if there is multiple category using css icons`